devices/tuya.js

```diff
@@ -426,7 +426,12 @@ module.exports = [
         model: 'TS0044',
```

Comment:
You probably need to add a separate entry with this `fingerprint`. Otherwise existing `TS0044` devices won't match any more.
Alternatively you could probably replace `zigbeeModel` with something like
```
fingerprint: [
  {modelID: 'TS0043'},
  {modelID: 'TS004F', manufacturerName: '_TZ3000_xabckq1v'},
],
```
But not sure if you can have fingerprint with just `modelID` without manufacturer.